docs-src/getting-started.md

```diff
@@ -0,0 +1,55 @@
+# Getting started with Services APIs
+
+## Installing CRDs
+
+Service APIs is a collection of Kubernetes Custom Resource Definitions that need
+to be installed into any Kubernetes cluster.
+
+To install the CRDs, please execute:
+
+```
+kubectl kustomize "github.com/kubernetes-sigs/service-apis/config/crd?ref=master" \
+| kubectl apply -f -
+```
+
+## Install an implementation
+
+Service APIs is a set of specification that multiple implementations confirm to.
+You will need to either install an implementation or verify if one is already
+setup for your cluster.
+
+## Sample Gateway
+
+Once you have the CRDs and an implementation installed, you are ready to
+use Service APIs.
+
+In this example, we are installing three resources:
+
+- An `acme-lb` GatewayClass which is being managed by a `acme.io/gateway-controller` controller running in the cluster.
+- A Gateway which is of type `acme-lb`:
+    - This gateway has a single HTTP listener on port 80 which selects HTTPRoutes
+      from all namespaces which have the label `app: foo` on them.
+
+- Finally, we have an HTTPRoute resource which is attached to the above Gateway
+  and has two rules:
+    - All requests with Path beginning with `/bar` are forwarded to my-service1
```

Comment:
Nit: I'm not sure path needs to be capitalized here. I think it could use an `a` before it though.